nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -140,6 +166,35 @@ public void isQueryEndpointCanSwitchToES() {
         }
     }
 
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.collections.core:OSGI-INF/collection-pageprovider-contrib.xml")
+    public void iCanUseFulltextOperatorWithElasticsearchPageProvider() {
+        PageProviderService pps = Framework.getService(PageProviderService.class);
+        PageProviderDefinition ppdef = pps.getPageProviderDefinition("default_collection");
+
+        HashMap<String, Serializable> props = new HashMap<>();
+        props.put(ElasticSearchNativePageProvider.CORE_SESSION_PROPERTY, (Serializable) session);
+        PageProvider<?> pp = pps.getPageProvider(ppdef.getName(), ppdef, null, null, null, 0L, props, "testCo",
+                session.getPrincipal().getName());
```

Comment:
Either check something on the `pp` (no collection?) here, or just initialize it after the creation of the collections (instead of `pp.refresh()`).

Reply:
I swapped the code blocks.